app/main/main_window.js

```diff
@@ -1,5 +1,11 @@
 "use strict";
 
+const CONFIG = {
```

Comment:
If we are going to use a set of constants for config values, this should be in common/AppConfig so it can be used by all the places of the app

Reply:
You mean using constants as `AppConfig.WINDOW_SIZE`, etc?

Reply:
that or AppConfig.configValues.WINDOW_SIZE

whatever you think best

Reply:
Yup makes sense, having `AppConfig.CONFIG.WINDOW_SIZE` would make sense as we'd not have to export anything else apart from AppConfig. I'll update this PR. :+1: